source/best-practices/security-bp.rst

```diff
@@ -65,7 +65,7 @@ censor the password before sending it, as done below.
 
   OS_AUTH_URL=https://api.cloud.catalyst.net.nz:5000/v3
```

Comment:
Change to "https://api.nz-por-1.catalystcloud.io:5000/"

Reply:
updated this example and others to remove the v3 and to have the catalystcloud.io change